bitacross-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -512,15 +524,13 @@ pub enum BitacrossRequestError {
 	Other(Vec<u8>),
 }
 
-async fn request_bit_across_inner(params: Params) -> Result<RpcReturnValue, Vec<u8>> {
-	let payload = get_request_payload(params)?;
-	let request = AesRequest::from_hex(&payload)
-		.map_err(|e| format!("AesRequest construction error: {:?}", e))?;
+async fn bitacross_task_create_inner(params: Params) -> Result<RpcReturnValue, Vec<u8>> {
+	let request = get_request_from_params(params)?;
 
 	let bit_across_request_sender = BitAcrossRequestSender::new();
 	let (sender, receiver) = oneshot::channel::<Result<BitAcrossProcessingResult, Vec<u8>>>();
 
-	bit_across_request_sender.send(BitAcrossRequest { sender, request })?;
+	bit_across_request_sender.send(BitAcrossRequest::CreateSignTask(request, sender))?;
```

Comment:
Currently this API is able to handle any `DirectCall`, why are you limiting it to 'signing tasks only' ? 
This was general api for submiting bitacross tasks - DirectCalls.
Now it's being coupled to 'signing' tasks only which is in my opinion a step back.

Please keep it as it is - or introduce new method for handling signing tasks only.

Reply:
> Optimizing ceremony data sharing logic, now it sends response back once handled by websocket server

There are two kinds of `DirectCall`, one is for submitting request from client (still `bitacross_submitRequest`), one is for communication between workers(now 'bitacross_btcDataShare').
You can check `bitacross_task_create_inner` and `bitacross_data_share_inner`, they are handled in different ways, the former waits for response sent from command processing thread, the latter not.
I think it's better to make them decoupled

Reply:
I added `CeremonyRoundCall` to make this clearer